bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/i18n/mielecloud.properties

```diff
@@ -1,6 +1,6 @@
-# Binding related texts
-binding.mielecloud.name=Miele@home Cloud Binding
-binding.mielecloud.description=This is the cloud-based Miele@home binding.
+# add-on related texts
+addon.mielecloud.name=Miele@home Cloud Binding
+addon.mielecloud.description=This is the cloud-based Miele@home addon.
```

Comment:
```suggestion
addon.mielecloud.description=This is the cloud-based Miele@home binding.
```

Reply:
I don't think this should be changed. It's still a binding. The whole idea of the core change is to extend the metadata to the other add-on types besides a binding, not to replace "binding" by "add-on".

Reply:
That's why I suggest to change it back. 🙂